helm/ffc-demo-payment-service/templates/service.yaml

```diff
@@ -1,13 +1,13 @@
-{{- if .Values.postgresExternalName }}
+{{- if quote .Values.postgresExternalName }}
 kind: Service
 apiVersion: v1
 metadata:
   name: {{ .Values.postgresHost }}
   labels:
-    environment: {{ .Values.environment }}
+    environment: {{ quote .Values.environment }}
 spec:
   type: ExternalName
-  externalName: {{ .Values.postgresExternalName }}
+  externalName: {{ quote .Values.postgresExternalName }}
```

Comment:
I'm not sure why this would be the case but the Jenkins error suggests quoting the `externalName` value results in an invalid deployment as it sees the quotes as part of the value.